packages/plugins/src/plugins/renzo/helpers.ts

```diff
@@ -0,0 +1,66 @@
+import { EvmNetworkIdType } from '@sonarwatch/portfolio-core';
+import { RenzoContractConfig, RenzoStakedContractConfig } from './types';
+import { getEvmClient } from '../../utils/clients';
+import {
+  activeStakeAbi,
+  getOutstandingWithdrawRequestsAbi,
+  withdrawRequestAbi,
+} from './abis';
+import { Address } from 'viem';
+
+export async function generateActiveStakeElement(
+  activeStakeContract: RenzoStakedContractConfig,
+  owner: string,
+  networkId: EvmNetworkIdType
+): Promise<bigint | undefined> {
+  const { address, token } = activeStakeContract;
+
+  const client = getEvmClient(networkId);
+  const activeStake = await client.readContract({
+    address,
+    abi: activeStakeAbi,
+    functionName: 'activeStake',
+    args: [owner as Address],
```

Comment:
Let's use stronger typing for our internal methods; casting params/variables using `as ` it typically a _smell_ that should be avoided.

```suggestion
  owner: Address,
  networkId: EvmNetworkIdType
): Promise<bigint | undefined> {
  const { address, token } = activeStakeContract;

  const client = getEvmClient(networkId);
  const activeStake = await client.readContract({
    address,
    abi: activeStakeAbi,
    functionName: 'activeStake',
    args: [owner],
```